migrations/upgrades/20200303111750_add_auth_ttl_field.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+use Phinx\Migration\AbstractMigration;
+
+class AddAuthTtlField extends AbstractMigration
+{
+    public function up()
```

Comment:
There's one thing I don't have clear. When I create a new Directus project, this migration is not applied (therefore I don't see the widget in the Directus backoffice. I tried also using the `change()` method to no avail.

Any hint on how to fix this?

Reply:
There is a folder of seeders too, you'll have to add the field there as well.